CHANGELOG.md

```diff
@@ -3,11 +3,29 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.9.0] - Unreleased
+
+### Added
+
+- Added `RequestSharedSecret`, `RequestPublicKey`, `RequestSecretKey`, `RequestKeyFactory` as wrappers for underlying Curve 25519 key functionality. ([#54])
+- Added Rust `pre-commit` hooks for repos. 
```

Comment:
Please add the `([#54])` refs to other items (and fix the one that says `([54])`)